### ISSUE TITLE
We need to gracefully handle closed indices

### ISSUE THREAD

**Issue by portante** [distributed-system-analysis/pbench#1027]
Ignoring the fact that we have a bad date stamp in the index name, we need to handle these kinds of errors when an index is enountered that is closed.

```
run-2019-01-01T01:08:02-UTC: EC2::ip-10-0-202-145/pbench-user-benchmark__2018.12.31T15.47.22: success
   2019-01-01T01:10:00-UTC: Finished /pbench/archive/fs-version-001/EC2::ip-10-0-202-145/TO-INDEX/pbench-user-benchmark__2018.12.31T15.47.22.tar.xz (size 43364992)
   2019-01-01T01:10:00-UTC: Starting /pbench/archive/fs-version-001/b06-h31-1029p/TO-INDEX/fio_glusterfs-HDD-25GBper-pod-seq_write_read_2018.12.31T18.55.51.tar.xz (size 74338724)
        done templates (end ts: 2019-01-01T01:11:02-UTC, duration: 2.43s, successes: 3, retries: 0)
es_index: ERROR KeyError('create',)
es_index: WARNING - retrying action
{
    "index": {
        "_index": "dsa.pbench.result-data.1970-01-01",
        "_type": "pbench-result-data",
        "_id": "fba255dc184516b9a52d395cb3253008",
        "status": 403,
        "error": "IndexClosedException[[dsa.pbench.result-data.1970-01-01] closed]"
    }
}
```

**Comment by portante** [comment]
See also #1026.

**Comment by portante** [comment]
Fixed by #1028.
